frontend/src/views/instructors/index.js

```diff
@@ -12,10 +12,30 @@ import {
     ActionHeader,
 } from "../../components/action-buttons";
 import { ContentArea } from "../../components/layout";
+import { ConnectedDeleteInstructorDialog } from "./delete-instructor-dialog";
+
+const BLANK_INSTRUCTOR = {
+    first_name: "",
+    last_name: "",
+    email: "",
+    utorid: "",
+    id: 0,
+};
 
 export function AdminInstructorsView() {
     const [addDialogVisible, setAddDialogVisible] = React.useState(false);
     const [inDeleteMode, setInDeleteMode] = React.useState(false);
+    const [deleteDialogVisible, setDeleteDialogVisible] = React.useState(false);
+    const [instructorToDelete, setInstructorToDelete] = React.useState(
+        BLANK_INSTRUCTOR
```

Comment:
What? Why....? Use `null` to initialize and do appropriate validation checks before actually deleting.